ogc-example/src/main/resources/application-spacetimekey.conf

```diff
@@ -190,13 +190,7 @@ layers = {
     type = "simplesourceconf"
     name = "spacetimetest"
     title = "Landsat 8 OGC Temporal SpaceTimeKey Test"
-    source = {
-      type = "geotrellis"
-      catalog-uri = "s3://geotrellis-test-non-public/spacetimekey-test"
-      layer = "spacetimekey-test"
-      zoom = 14
-      band-count = 1
-    }
+    source = "gt+s3://geotrellis-test-non-public/spacetimekey-test?layer=spacetimekey-test&zoom=14&band_count=1"
```

Comment:
Also it looks like that `ogc-example/src/main/resources/application.conf` has to be changed and tested before the merge.

Reply:
Yup. Addressed.